.env.example

```diff
@@ -1,2 +1,4 @@
+# your Supabase url, e.g. https://wfzpewmlyiozupulbuur.supabase.co
 SUPABASE_URL=""
-SUPBASE_ANON_KEY=""
```

Comment:
Is this a typo in our template? If so please push a fix.

Reply:
It is specific to this repository, there is no such value in the template.